src/components/layout/drawer/Drawer.jsx

```diff
@@ -99,7 +99,7 @@ export default function Sidebar({
               || category === 'pokestops'
               || category === 'nests')
               && (
-                <Grid item xs={6} style={{ textAlign: 'center' }}>
+                <Grid item xs={7} style={{ textAlign: 'center' }}>
```

Comment:
I'm not a huge fan of this one as it just looks off centered in non-Polish languages. Maybe some logic specific to Polish language or preferably, something to shrink the size of the font to fit the xs={6} button size.

Reply:
Per-language would be better... but it's a PITA 😂I will remove changes related to grid for now.